packages/embark/src/lib/modules/ens/ENSFunctions.js

```diff
@@ -101,7 +101,7 @@ function resolveName(name, ens, createResolverContract, callback, _namehash) {
 }
 
 if (typeof module !== 'undefined' && module.exports) {
-  module.exports = {
+  exports.default = {
```

Comment:
Does this have to do with the babel transform and ES6 module expectations?

I wondering if a transform-runtime setting might smooth it out, but not really sure. See: https://github.com/embark-framework/embark/blob/master/packages/embark/src/lib/modules/pipeline/webpack.config.js#L145.

Reply:
Yeah. Babel transforms `export default` into `exports.default` in `embarkjs.js`, but if `module.exports` is present, like is the case here, it overrides `exports.default.

I'll try to use the plugin to see if it works.

Reply:
I tried and it didn't work. Checked the options and it doesn't look like there's something there that'll help.

Maybe you have another idea, I tried googling, but didn't find a solution.

Reply:
Okay, another option might be to refactor `ENSFunctions.js` to use ES6 syntax. So this part:
```
if (typeof module !== 'undefined' && module.exports) {
  exports.default = {
    registerSubDomain,
    resolveName,
    lookupAddress
  };
}
```
Would change to
```
export registerSubDomain;
export resolveName;
export lookupAddress;
```

Or, I think you could just put the `export` before the function declarations you want to export.

In `modules/ens/index.js` you should be able to `import {lookupAddress, registerSubDomain, resolveName} from './ENSFunctions'` because everything in `package/embark/src` is getting run through babel.

There might be some additional refactoring needed with this approach, so not sure it's worth it compared to the working solution you came up with.

Reply:
Nice work on the changes to get rid of the `.default` thing!